website/docs/r/app_service.html.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service"
+sidebar_current: "docs-azurerm-resource-app-service-x"
+description: |-
+  Manages an App Service (within an App Service Plan).
+
+---
+
+# azurerm\_app\_service
+
+Manages an App Service (within an App Service Plan).
+
+## Example Usage
```

Comment:
I would really like to see 1 or 2 other example usages here - something maybe a little more complex with a connection_string struct maybe?

Reply:
updated this to be a `.net 4.x` example with a sql connection string, and added a new java example